Brokerages/Zerodha/ZerodhaBrokerage.cs

```diff
@@ -626,7 +620,10 @@ public override bool UpdateOrder(Order order)
                 UnlockStream();
                 return false;
             }
-
+            if (orderProperties.Exchange == null)
+            {
+                throw new NullReferenceException("Please set exchange attribute using DefaultOrderProperties because placing an order");   
```

Comment:
Think we can join this check with the one above? which I think isn't logging the right error message `Invalid product type` but it's checking for `ZerodhaOrderProperties`, same for `PlaceOrder`

Reply:
If we will merge the checks then we may not able to log specific errors? 

```
if (orderProperties == null or orderProperties.Exchange == null )
{
log
}
```